bitrise.yml

```diff
@@ -881,6 +881,21 @@ workflows:
         - scheme: Fennec
         - simulator_device: iPhone 8
     - deploy-to-bitrise-io@1.9: {}
+    - script@1:
```

Comment:
No need to add it here since this belongs to the RunUnitTestsNewL10N workflow that is going to be removed as part as the bitrise clean up :) 

Can you add it though to the other workflows we use: 
- [CommonBuild-UItest-XCUISmoketest](https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L2131)
- [RunAllXCUITests](https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L1729)
- [L10nBuild](https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L379)
- [NewXcodeVersions](https://github.com/mozilla-mobile/firefox-ios/blob/main/bitrise.yml#L2891)

Thanks for finding this workaround. Hopefully that will fix all the builds 🤞